.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -16,41 +19,53 @@
 this_script_dir = pathlib.Path(__file__).parent
 repo_root = this_script_dir.parent
 
-helm_repo_name = "temp-stackrox-oss-repo-should-not-see-me"
+HELM_REPO_NAME = "temp-stackrox-oss-repo-should-not-see-me"
```

Comment:
Perhaps even
```suggestion
_HELM_REPO_NAME = "temp-stackrox-oss-repo-should-not-see-me"
```
if it's not used from elsewhere?

Reply:
To a large extent I did what style or lint asked. I prefer a leading `_` for internals but that is probably best applied as a lint rule to another PR (with all the necessary fixes).